### PR TITLE
Add Architect, Update Inventory, Fix README

### PR DESCRIPTION
**Pull request by ColonelPanics**

### CODE REVIEW

File: tools/flight-architect.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# OpenFlight Tools. If not, see:
+#
+#  https://creativecommons.org/licenses/by-sa/4.0/
+#
+# For more information on OpenFlight Tools, please visit:
+# https://github.com/openflighthpc/openflight-tools
+#===============================================================================
+---
+:name: architect
+:url: https://github.com/alces-software/underware/archive/0.6.0-rc4.zip
```
Comment (mjtko) [inline tools/flight-architect.yml]: This should probably point to the openflighthpc/flight-architect repo.

**Comment by mjtko** [comment]
Apart from the minor issue of the URL mentioned above, LGTM!

File: tools/flight-architect.yml (hunk 1 above)
Comment (ColonelPanics) [reply tools/flight-architect.yml]: Oops! Had added this before migrating Underware to Flight Architect. Sorted now.

**Comment by ColonelPanics** [comment]
@mjtko if you're happy to merge can we get this through and a new RPM created?

**Pull request merged**